packages/flutter_tools/lib/src/test/flutter_web_platform.dart

```diff
@@ -104,7 +96,19 @@ class FlutterWebPlatform extends PlatformPlugin {
     );
   }
 
-  Uri testUri;
+  /// Parsed package map from `.packages` of the current working directory.
+  final PackageMap _packageMap = PackageMap(p.join(p.current, '.packages'));
```

Comment:
Nit: use package_config/discovery.dart instead, since that will automatically handle finding .dart_tool/package_config.json when we enable NNBD:

https://github.com/flutter/flutter/blob/e340c19905fd2eca86be3c429bf659963ed9d6f0/packages/flutter_tools/test/general.shard/web/devfs_web_test.dart#L44